doc/api/cli.md

```diff
@@ -154,53 +208,80 @@ Note: v8 options allow words to be separated by both dashes (`-`) or underscores
 For example, `--stack-trace-limit` is equivalent to `--stack_trace_limit`.
 
 ### `--tls-cipher-list=list`
+<!-- YAML
+added: v4.0.0
```

Comment:
@Trott +1 for keeping it as 4.0.0… the doctool would support adding multiple versions here, but I implemented that more with LTS backports in mind, and I think adding some 0.12.x here would be more confusing than helping.